autogen/v1.9.json

```diff
@@ -1,4 +1,7 @@
 {
+    "GNDSDir": "test",
```

Comment:
Yes; I'll change it.

Note that we're actually using prototype.json, not v1.9.json, to generate the small "GNDS v1.9" prototype codes that are currently in the repo. Therefore, we should currently only be running `json2class.exe prototype.json`. The `v1.9.json` file is only going to be relevant if and when we fix up the weirdnesses in all of those `summary_*.json` files, and then also write tests and such for all of those. I anticipate that we may eventually want to refactor the `autogen/` contents, so that our prototype stuff appears, in some form, in a proper `autogen` `test/` directory. Then, we may remove the "v1.9" business entirely and go directly to GNDS 2.0. This is all another discussion for another time.